src/Repository/QueueItemRepositoryInterface.php

```diff
@@ -15,4 +16,6 @@ interface QueueItemRepositoryInterface extends RepositoryInterface
     public function findAllToImport(): array;
 
     public function findOneToImport(string $akeneoEntity, string $akeneoIdentifier): ?QueueItemInterface;
+
+    public function findToCleanup(DateTime $dateLimit): array;
```

Comment:
Adding the method to the interface is a BC break:
```
[BC] ADDED: Method findToCleanup() was added to interface Webgriffe\SyliusAkeneoPlugin\Repository\QueueItemRepositoryInterface
```
We have to figure out how to deal with it.